commons/src/main/java/org/jetbrains/bsp/bazel/commons/FileUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.bsp.bazel.commons;
+
+import com.google.common.io.CharSource;
+import com.google.common.io.Files;
+import io.vavr.control.Try;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+
+public final class FileUtils {
+
+  public static String readFileContentOrEmpty(Path filePath) {
+    return Try.success(filePath)
+        .map(Path::toFile)
+        .map(file -> Files.asCharSource(file, Charset.defaultCharset()))
+        .mapTry(CharSource::read)
+        .getOrElse("");
```

Comment:
Maybe deal with exceptions in an exception handler instead of silently swallowing them. You could log them for example.

Reply:
I'm working on that